src/utils.js

```diff
@@ -65,10 +66,24 @@ const getPackageJson = async () => {
   }
 }
 
+/**
+ * Check if given file path is valid
+ */
+const validateFilePath = filePath => {
+  let isValid = null
+  try {
+    isValid = fs.lstatSync(filePath).isFile()
+  } catch (err) {
+    isValid = false
```

Comment:
This could the place to display an error message if template is not valid and abort process